WBooks/LibraryTableViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  LibraryTableViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 20/09/2022.
+//
+
+import UIKit
+
+class LibraryTableViewController: UIViewController {
+
+    //MARK: Properties
+    
+    @IBOutlet weak var libraryTableView: UITableView!
+    var library : [Library]?
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        libraryTableView.delegate = self
+        libraryTableView.dataSource = self
+        loadSampleLibrarys()
+        
+    }
+
+    
+     
+     //MARK: Private Methods
+      
+    private func loadSampleLibrarys() {
+         
+         let photo1 = UIImage(named: "Cover1")
+         let photo2 = UIImage(named: "Cover2")
+         let photo3 = UIImage(named: "Cover3")
+         let photo4 = UIImage(named: "Cover4")
+         let photo5 = UIImage(named: "Cover5")
```

Comment:
align the variables of the images, they have one more space compared to the ones below